### PR TITLE
Sort on repo size in admin panel

### PR DESCRIPTION
**Pull request by drsect0r**
Fix for #1653

Constructive feedback is appreciated as this is my first pull request.

### CODE REVIEW

**Comment by lunny** [comment]
LGTM

File: options/locale/locale_en-US.ini [hunk 1]
```diff
@@ -666,6 +666,8 @@ issues.label_deletion_desc = Deleting this label will remove its information in
 issues.label_deletion_success = This label has been deleted successfully!
 issues.label.filter_sort.alphabetically = Alphabetically
 issues.label.filter_sort.reverse_alphabetically = Reverse alphabetically
+issues.label.filter_sort.by_size = Size
+issues.label.filter_sort.reverse_by_size = Reverse Size
```
Comment (sapk) [inline options/locale/locale_en-US.ini]: I would prefer not use capital `S` for `Reverse Size` and follow format like `Reverse alphabetically`

**Comment by sapk** [comment]
Except for my little comment LGTM

**Comment by drsect0r** [comment]
@lunny @sapk Thanks! I have pushed the last changes should I rebase or is this good-to-go?

**Review by appleboy — APPROVED** [review]

**Comment by appleboy** [comment]
LGTM

**Pull request merged**